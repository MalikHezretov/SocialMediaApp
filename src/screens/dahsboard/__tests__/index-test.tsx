import 'react-native';
import React from 'react';
import Dashboard from '..';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import store from '../../../redux/store';

it('renders Dashboard correctly', () => {
  const testRenderer = renderer.create(
    <Provider store={store}>
      <Dashboard />
    </Provider>,
  );

  expect(testRenderer).toMatchSnapshot();
});
