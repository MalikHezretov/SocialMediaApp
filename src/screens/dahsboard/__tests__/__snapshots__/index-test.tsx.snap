// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Dashboard correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTScrollView
    ListFooterComponent={<ActivityIndicator />}
    data={Array []}
    getItem={[Function]}
    getItemCount={[Function]}
    initialNumToRender={10}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onEndReached={[Function]}
    onEndReachedThreshold={0.1}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onRefresh={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    refreshControl={
      <RefreshControlMock
        onRefresh={[Function]}
        refreshing={false}
      />
    }
    refreshing={false}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsVerticalScrollIndicator={false}
    stickyHeaderIndices={Array []}
    viewabilityConfigCallbackPairs={Array []}
  >
    <RCTRefreshControl />
    <View>
      <View
        onLayout={[Function]}
      >
        <ActivityIndicator />
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
